validphys2/src/validphys/commondataparser.py

```diff
@@ -362,6 +348,26 @@ def apply_label(self, var, value):
 
 ### kinematics end
 
+### Variants
+@dataclasses.dataclass(frozen=True)
+class Variant:
+    """The new commondata format allow the usage of variants
+    A variant can overwrite a number of keys, as defined by this dataclass
+    """
+
+    data_uncertainties: Optional[list[ValidPath]] = None
+    theory: Optional[TheoryMeta] = None
+    data_central: Optional[ValidPath] = None
+    kinematic_coverage: Optional[list] = None
+    kinematics: Optional[ValidKinematics] = None
+    tables: Optional[list] = None
+    ndata: Optional[int] = None
```

Comment:
Hi @giacomomagni let's not do this one. If the number of datapoints actually changes, that's a completely different dataset.

Actually, I would say the same is true for `kinematic_coverage` and `plotting`. Those are enough to say they are a different dataset (and, tbh, being the same dataset it could be recasted to the same kinematic variables, so only the numerical values should change)

Reply:
Okay, it was just an attempt, i'll revert it.
So then we go for new datasets, we can include `_HEPDATA_` in the name to distinguish it.

Reply:
Let me invoke the master @enocera 

I'm not sure whether maybe we want to change the _old ones_ to be called `whatever....-legacy` so that the new ones have the "easy name"?

Reply:
So renaming old dataset will be not backward compatible but it's also possible, I mean in the long terms you'd like to get rid of them.

Reply:
By putting the name of on the `dataset_names.yml` all fits using the (very) old name will be reproducible just the same, and the new ones (the ones that @RoyStegeman is collecting in the wiki) will now refer to the hepdata dataset which is good I'd say (_if_ we consider that the hepdata dataset is the final truth, of course)

Reply:
I see, okay than we can add ``_LEGACY_`` to the old datsets and proceed with the new one. 
I'll do it asap.

Reply:
I confirm we should go for a new data set (as I mentioned at the beginning of the thread). I validate @scarlehoff's proposal.

Reply:
To support full backward compatibility  with the current studies using the new commondata, 
it's better to change the name of the new dataset.